[2024-06-24T02:10:43.534+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-24T02:10:43.552+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: encryption_dag.encrypt_and_upload manual__2024-06-24T02:10:17.658430+00:00 [queued]>
[2024-06-24T02:10:43.562+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: encryption_dag.encrypt_and_upload manual__2024-06-24T02:10:17.658430+00:00 [queued]>
[2024-06-24T02:10:43.563+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-06-24T02:10:43.584+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): encrypt_and_upload> on 2024-06-24 02:10:17.658430+00:00
[2024-06-24T02:10:43.601+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=490) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-06-24T02:10:43.603+0000] {standard_task_runner.py:63} INFO - Started process 503 to run task
[2024-06-24T02:10:43.609+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'encryption_dag', 'encrypt_and_upload', 'manual__2024-06-24T02:10:17.658430+00:00', '--job-id', '179', '--raw', '--subdir', 'DAGS_FOLDER/extraer_encriptar.py', '--cfg-path', '/tmp/tmpktvfy6_r']
[2024-06-24T02:10:43.612+0000] {standard_task_runner.py:91} INFO - Job 179: Subtask encrypt_and_upload
[2024-06-24T02:10:43.669+0000] {task_command.py:426} INFO - Running <TaskInstance: encryption_dag.encrypt_and_upload manual__2024-06-24T02:10:17.658430+00:00 [running]> on host f4301f0367ea
[2024-06-24T02:10:43.785+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='encryption_dag' AIRFLOW_CTX_TASK_ID='encrypt_and_upload' AIRFLOW_CTX_EXECUTION_DATE='2024-06-24T02:10:17.658430+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-06-24T02:10:17.658430+00:00'
[2024-06-24T02:10:43.787+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-24T02:10:43.812+0000] {extraer_encriptar.py:41} INFO - Starting encryption process...
[2024-06-24T02:10:43.827+0000] {connection.py:274} WARNING - Connection schemes (type: google_cloud_platform) shall not contain '_' according to RFC3986.
[2024-06-24T02:10:43.830+0000] {base.py:84} INFO - Using connection ID 'google_cloud_default' for task execution.
[2024-06-24T02:10:45.708+0000] {extraer_encriptar.py:48} INFO - File downloaded from GCS: datasets/athletes.csv
[2024-06-24T02:10:48.432+0000] {gcs.py:574} INFO - Data stream uploaded to datasets/athletes.csv.enc in golden-cheetah-datasets bucket
[2024-06-24T02:10:48.433+0000] {extraer_encriptar.py:55} INFO - File uploaded to GCS: datasets/athletes.csv.enc
[2024-06-24T02:10:48.848+0000] {gcs.py:698} INFO - Blob datasets/athletes.csv deleted.
[2024-06-24T02:10:48.849+0000] {extraer_encriptar.py:59} INFO - Original file deleted from GCS: datasets/athletes.csv
[2024-06-24T02:10:48.849+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-06-24T02:10:48.850+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-24T02:10:48.865+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=encryption_dag, task_id=encrypt_and_upload, run_id=manual__2024-06-24T02:10:17.658430+00:00, execution_date=20240624T021017, start_date=20240624T021043, end_date=20240624T021048
[2024-06-24T02:10:48.903+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-06-24T02:10:48.919+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-06-24T02:10:48.923+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
